app/models/person_escort_record.rb

```diff
@@ -1,8 +1,13 @@
 class PersonEscortRecord < VersionedModel
+  PERSON_ESCORT_RECORD_NOT_STARTED = 'not_started'.freeze
+  PERSON_ESCORT_RECORD_IN_PROGRESS = 'in_progress'.freeze
+  PERSON_ESCORT_RECORD_COMPLETED = 'completed'.freeze
+  PERSON_ESCORT_RECORD_CONFIRMED = 'confirmed'.freeze
```

Comment:
maybe we can avoid heading all this constant with `PERSON_ESCORT_RECORD_` since they are scoped into the `PersonEscortRecord` class?

Reply:
This seems to be the convention in all other models, also I think it's best to prepend the name of the model before the constant in this case as we might have other models with the similar constants in the future, so using the model name allows values to be explicit and scoped